redash/settings/__init__.py

```diff
@@ -30,6 +31,12 @@
 CELERY_RESULT_EXPIRES = int(os.environ.get(
     "REDASH_CELERY_RESULT_EXPIRES",
     os.environ.get("REDASH_CELERY_TASK_RESULT_EXPIRES", 3600 * 4)))
+CELERY_SSL_CONFIG = {
```

Comment:
@rauchy how does this look as far as settings?

I'll try this change out on our self-hosted setup later today

Reply:
Settings look fine to me. However, the build is failing and requires your attention.